edx_notifications/static/js/HelloWorld.js

```diff
@@ -0,0 +1,3 @@
+function helloWorld(){
+    return "Hello world";
+}
```

Comment:
I'd really not to have to add a "HelloWorld.js" file into the codebase, just to verify that we can launch a Jasmine test. Any chance we can get a bare bones test of one aspect of the existing Backbone code in server/web/static/js?